test/e2e_node/image_list.go

```diff
@@ -52,12 +52,18 @@ var NodeImageWhiteList = sets.NewString(
 	imageutils.GetE2EImage(imageutils.Netexec),
 	imageutils.GetE2EImage(imageutils.Nonewprivs),
 	imageutils.GetPauseImageNameForHostArch(),
-	framework.GetGPUDevicePluginImage(),
 )
 
 func init() {
 	// Union NodeImageWhiteList and CommonImageWhiteList into the framework image white list.
 	framework.ImageWhiteList = NodeImageWhiteList.Union(commontest.CommonImageWhiteList)
+
+	// parse the device plugin image from url
+	if image, err := framework.GetGPUDevicePluginImage(); err != nil {
+		glog.Errorf("Failed to parse the device plugin image: %v", err)
```

Comment:
Can you log the errors in `GetGPUDevicePluginImage()` itself instead of returning it and then logging it here?

Reply:
@mindprince sure. I was just afraid that would import a new package in the gpu_list.go, which I'm not sure if causes the loop of imports.